platinum.us/main.lsf

```diff
@@ -1720,6 +1720,6 @@ Overlay overlay120
 Overlay overlay121
```

Comment:
If you choose to name the overlay, make sure that you also do so here, changing `overlay121` to `dworld_warp` (or whatever you choose to name it).